cms/djangoapps/contentstore/rest_api/v1/serializers/vertical_block.py

```diff
@@ -181,3 +181,10 @@ class ContainerChildrenSerializer(serializers.Serializer):
     is_published = serializers.BooleanField()
     can_paste_component = serializers.BooleanField()
     display_name = serializers.CharField()
+    upstream_ready_to_sync_children_info = serializers.ListField(
+        child=serializers.DictField(
+            child=serializers.CharField()
+        ),
```

Comment:
This converts all fields to string including boolean field, so even "False" is considered [`true` in frontend](https://github.com/openedx/frontend-app-authoring/pull/2516/files#diff-ab8d53e2e6b471f8f9b1462cf1f3e3d4469111dd86431a28a130dd103b3aee83R203). 

<img width="596" height="183" alt="Image" src="https://github.com/user-attachments/assets/63da9875-b799-420f-9fc1-a92585ef6414" />

Please use a concrete serializer class for children data here.